packages/run-protocol/src/reserve/doc.md

```diff
@@ -0,0 +1,129 @@
+# Reserve 
+
+### addIssuer
+
+```mermaid
+sequenceDiagram
+
+participant caller
+participant addIssuer
+participant ammPublicFacet
+participant brandForKeyword
+participant keywordForBrand
+participant zcf
+
+
+caller ->> addIssuer: addIssuer(inIssuer, inKeyword)
+addIssuer ->> addIssuer: get brand from inIssuer
+addIssuer ->> ammPublicFacet: get liquidity issuer for inIssuer
```

Comment:
```suggestion
addIssuer ->> ammPublicFacet: get liquidity issuer for issuer's brand
```
Though I'm not clear on what a "liquidity issuer" is. @Chris-Hibbert would know.

Reply:
Providers of liquidity to a pool in the AMM get Liquidity tokens (represented by a unique ERTP issuer) corresponding to the secondary currency in that pool. The Reserve needs to know about that because it's providing liquidity, and will get the corresponding liquidity tokens. The diagram looks correct to me.